esx_service/utils/auth_data.py

```diff
@@ -220,7 +226,7 @@ def remove_vms(self, conn, vms):
     def replace_vms(self, conn, vms):
         """ Remove vms from the vms table for this tenant. """
```

Comment:
comment is wrong - cut-n-pasted obviously. 
Also, do we really need "replace" or do we need "update" ?